internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -112,6 +114,10 @@ func (e *EnvStackConfig) Template() (string, error) {
 
 // Parameters returns the parameters to be passed into a environment CloudFormation template.
 func (e *EnvStackConfig) Parameters() ([]*cloudformation.Parameter, error) {
+	httpsListener := "false"
+	if len(e.in.ImportCertARNs) != 0 || e.in.App.DNSDelegationRole() != "" {
```

Comment:
question 🤚🏼  Why do we use `DNSDelegationRole` to determine whether it's delegated or not? Is there a more straightforward way for us to tell 🤔 ?